docs/user-guide/administration/backup-restoration.md

```diff
@@ -0,0 +1,158 @@
+| --database | No | greptime-* | Name of the database to export |
+| --export-jobs, -j | No | 1 | Number of parallel export jobs |
+| --max-retry | No | 3 | Maximum retry attempts per job |
+| --target, -t | No | all | Export target (schema/data/all) |
+| --start-time | No | - | Start of time range for data export |
+| --end-time | No | - | End of time range for data export |
+| --auth-basic | No | - | Basic authentication credentials |
+
+### Export Targets
+- `schema`: Exports table schemas only (`SHOW CREATE TABLE`)
+- `data`: Exports table data only (`COPY DATABASE TO`)
+- `all`: Exports both schemas and data (default)
+
+### Output Directory Structure
+```
+<output-dir>/
+└── <catalog>/
+    └── <database>/
+        ├── create_database.sql
+        ├── create_tables.sql
+        ├── copy_from.sql
+        └── <data files>
+```
+
+## Import Tool
+
+### Command Syntax
+```bash
+greptime import [OPTIONS]
+```
+
+### Options
+| Option | Required | Default | Description |
+|--------|----------|---------|-------------|
+| --addr | Yes | - | Server address to connect |
+| --input-dir | Yes | - | Directory containing backup data |
+| --database | No | greptime-* | Name of the database to import |
+| --import-jobs, -j | No | 1 | Number of parallel import jobs |
+| --max-retry | No | 3 | Maximum retry attempts per job |
+| --target, -t | No | all | Import target (schema/data/all) |
+| --auth-basic | No | - | Basic authentication credentials |
+
+### Import Targets
+- `schema`: Imports table schemas only
+- `data`: Imports table data only
+- `all`: Imports both schemas and data (default)
+
+## Common Usage Scenarios
+
+### Full Databases Backup
+```bash
+# Export all databases(in default catalog) backup
+greptime export --addr localhost:4000 --output-dir /tmp/backup/greptimedb
+
+# Import all databases(in default catalog)
+greptime import --addr localhost:4000 --input-dir /tmp/backup/greptimedb
+```
+
+### Schema-Only Operations
+```bash
+# Export only schemas
+greptime export --addr localhost:4000 --output-dir /tmp/backup/schemas --target schema
+
+# Import only schemas
+greptime import --addr localhost:4000 --input-dir /tmp/backup/schemas --target schema
+```
+
+### Time-Range Based Backup
+```bash
+# Export data within specific time range
+greptime export --addr localhost:4000 \
+    --output-dir /tmp/backup/timerange \
+    --start-time "2024-01-01 00:00:00" \
+    --end-time "2024-01-31 23:59:59"
+```
+
+### Specific Catalog/Database Backup
+```bash
+# To export a specific catalog
+greptime export --addr localhost:4000 --output-dir /tmp/backup/greptimedb --database '{my_catalog_name}-*'
+# To export a specific database from the default catalog
+greptime export --addr localhost:4000 --output-dir /tmp/backup/greptimedb --database '{my_database_name}'
+# To export a specific database from a particular catalog
+greptime export --addr localhost:4000 --output-dir /tmp/backup/greptimedb --database '{my_catalog_name}-{my_database_name}'
```

Comment:
please add a description about the architecture between catalogs and databases to the overview section in this page.

Reply:
I'll remove catalog since there is only one named greptime.